scripts/dotnet.py

```diff
@@ -276,6 +278,7 @@ def build(self,
                     '--configuration', configuration,
                     '--framework', target_framework_moniker,
                     '--no-restore',
+                    "/p:NuGetPackageRoot={}".format(packages_path),
```

Comment:
for some reason, when we are doing `dotnet restore --packages $path` and then `dotnet build --no-restore` and using Arcade (which has some .targets and .props files that affect some settings) we need to pass the path to packages folder to the build command.

big thanks to @ViktorHofer for helping me with this and providing a solution